### DISCUSSION TITLE
Cannot file support requests under my pro plan

### DISCUSSION THREAD

**Discussion by NickAnthony** [vercel/community#4385] [Account Issue] [answered]
### Question

Context: My account under this email is on the default hobby plan.  This morning, I paid and upgraded to a Pro Plan.  I was forced to create a team, migrate my project/database over to the team account, and everything.

Problem: I tried to file a support ticket using my pro plan.  My personal account is a hobby plan (by necessity) but I'm on a Pro Plan team.  I tried to file a support ticket using the Pro Plan team account, but Vercel still thinks I'm on the hobby plan, so I'm getting redirected to the hobby plan help page!  So that is why I'm posting here - I cannot file support tickets even with a Pro Plan.

Why did I try to file support ticket?  Vercel says I'm using two databases, whereas I only have one.  Because the Pro Plan has a limit of one database, it's REFUSING connection until I upgrade AGAIN to the next tier (which is enterprise).

So now I've paid for the Pro Plan, but I cannot file support requests and I cannot update any databases, so my deployment is effectively frozen.  Please help me out Vercel team!

**Comment by amyegan** [comment]
Hey @NickAnthony. I'm also seeing the problem you reported on the support form. Thank you for reporting it! I'm reaching out the the team to investigate why that's happening so we can fix it ASAP.

Creating an additional database shouldn't require you to upgrade to an Enterprise plan. While a Pro plan includes 1 database at no cost, you currently have the option to create more databases on a Pro plan at an [additional cost of 1.00 USD per database](https://vercel.com/docs/storage/vercel-postgres/usage-and-pricing#pricing). 

Can you share a video or screenshots so I can see what you see regarding a refused connection until you upgrade to enterprise? I'm not aware of anything that would do that, so it sounds like a bug. If you can share the steps you took, that will give me a better idea of where to look for the problem.

**Reply by amyegan** [reply, marked as answer]
The support form on https://vercel.com/help is working correctly now. Be sure to select your Team as the affected account at the top of the form to enable the applicable Problem Area options. Thanks again for reporting that `/help` page problem!

**Comment by NickAnthony** [comment]
Thanks for getting to this so quickly!  I confirmed that I can now file
support requests!

For the database issue: it seems to have been resolved.

I created a second database and then deleted it, realizing I could transfer
over my existing database.  The dashboard for database usage stills shows a
yellow circle with a "2 databases used out of 1" - I have attached the
image of what I'm seeing.  My local setup *was* complaining that the
connection was refused due to database limits, but this morning it seems
back to normal.

I cannot reproduce it anymore, so this ticket is now resolved!

- Nick
